alwaysontop/helpers.js

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Could you please rename the file to functions.js? This way it will be more consistent with jitsi-meet file names.